src/pocketmine/entity/projectile/Arrow.php

```diff
@@ -74,6 +77,10 @@ public function onUpdate($currentTick){
 
 		if($this->onGround or $this->hadCollision){
 			$this->setCritical(false);
+			if($this->sound === true and $this->level !== null){ //Prevents error of $this->level returning null
```

Comment:
I found why it happens, the arrow is despawning too early so it causes this problem.